spring-data-vmware-gemfire/src/main/java/org/springframework/data/gemfire/listener/annotation/ContinuousQuery.java

```diff
@@ -29,6 +27,13 @@
 @Documented
 public @interface ContinuousQuery {
 
+	/**
+	 * Specifies event types to be excluded from the CQ.
+	 *
+	 * Defaults to empty.
+	 */
+	ExcludedEvent[] excludedEvents() default {};
```

Comment:
As per our discussion, can we please not directly reference these enums and use our own. This would even allow us to wrap them with different one's that we can come up with.

Reply:
I've now added an enum that wraps/abstracts ExcludedEvent